app/mailers/single_cell_mailer.rb

```diff
@@ -29,14 +29,22 @@ def notify_admin_upload_fail(study_file, error)
     end
   end
 
-  def notify_user_parse_complete(email, title, message)
+  def notify_user_parse_complete(email, title, message, study)
     @message = message
+    @study = study
```

Comment:
Where does this get used?

Reply:
In the views for the parse completion/failure emails.  There's a link to the study at the bottom, and I needed to pass it in so I can render the URL using the helper.

Reply:
got it, in the erb's